IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate.Tests/BaseTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace IntelliTect.TestTools.Selenate.Tests
+{
+    public class BaseTest
+    {
+        protected void TestVoidException()
```

Comment:
Consider a more explicit name. Something like `ThrowException`. Then the method below would be `ThrowExceptionWithReturn`